README.md

```diff
@@ -54,66 +55,255 @@ Any invalid request body parameters will result in a `422 Unprocessable Entity`
 
 Request:
 
-    POST http://{host}:{port}/api/{version}/users/{user_id}/locations/{location_id}/donations
-    
-    {
-        size: INT (none = 0 | small = 1 | medium = 2 | large = 3),
-        comments: String (max length 255 characters)
+        POST http://{host}:{port}/api/{version}/users/{user_id}/locations/{location_id}/donations
         
-    }
+        {
+            size: INT (none = 0 | small = 1 | medium = 2 | large = 3),
+            comments: String (max length 255 characters)
+            
+        }
     
  Response:
-    HTTP 201 CREATED
-    
-    [
-        {
-            id: Int,
-            size: Int (0..3)
-            location_id: Int,
-            user_id: Int,
-            created_at: DateTime,
-            updated_at: DateTime
-        },
-        ...
-    ] 
+ 
+        HTTP 201 CREATED
+        
+        [
+            {
+                id: Int,
+                size: Int (0..3)
+                location_id: Int,
+                user_id: Int,
+                created_at: DateTime,
+                updated_at: DateTime
+            },
+            ...
+        ] 
     
 
-### Updating a given Resource
+### Updating a given user's Donation
 **This route is authorized only for the owner of the Donation resource and administrators**
 
 This route will return a `204 No Content` on successful updates because it assumes the Client has the most up-to-date representation of the Donation resource. Any invalid request body parameters will result in a `422 Unprocessable Entity` along with an errors object detailing the invalid entity data.
 
 Request:
 
-    PUT http://{host}:{port}/api/{version}/users/{user_id}/locations/{location_id}/donations/{id}
-    
-    {
-        size: INT (none = 0 | small = 1 | medium = 2 | large = 3),
-        comments: String (max length 255 characters)
+        PUT http://{host}:{port}/api/{version}/users/{user_id}/locations/{location_id}/donations/{id}
         
-    }
+        {
+            size: INT (none = 0 | small = 1 | medium = 2 | large = 3),
+            comments: String (max length 255 characters)
+            
+        }
     
  Response:
-    HTTP 204 NO CONTENT
+ 
+        HTTP 204 NO CONTENT
```

Comment:
responses to PUTs are always the representation of the resource (basically the same json body as the POST create response), not no content.